web_responsive/tests/test_res_users.py

```diff
@@ -1,32 +1,27 @@
 # Copyright 2023 Taras Shabaranskyi
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+from odoo.tests import new_test_user
 
-from odoo.tests.common import TransactionCase
+from odoo.addons.base.tests.common import BaseCommon
 
 
-class TestResUsers(TransactionCase):
+class TestResUsers(BaseCommon):
     def test_compute_redirect_home(self):
-        record = self.env["res.users"].create(
-            {
-                "action_id": False,
-                "is_redirect_home": False,
-                "name": "Jeant",
-                "login": "jeant@mail.com",
-                "password": "jeant@mail.com",
-            }
+        record = new_test_user(
+            self.env,
+            login="jeant@mail.com",
         )
-
-        record._compute_redirect_home()
         self.assertFalse(record.is_redirect_home)
 
-        action_obj = self.env["ir.actions.actions"]
-        record.action_id = action_obj.create(
-            {"name": "Test Action", "type": "ir.actions.act_window"}
+        action = self.env["ir.actions.act_window"].create(
+            {
+                "name": "Test Action",
+                "type": "ir.actions.act_window",
+                "res_model": record._name,
+            }
         )
-        record._compute_redirect_home()
+        record.action_id = self.env["ir.actions.actions"].browse(action.id)
```

Comment:
Is there any reason to rebrowse?

```suggestion
        record.action_id = action.id
```

Reply:
According to Odoo's reference PR...

![image](https://github.com/user-attachments/assets/066b2212-7865-4bfd-aeb7-51aaf42ef6a9)

Reply:
OK, then the same kind of "comment" should be needed here for future reference.

Reply:
Changed, thank you. Yes, it is possible to do it without rebrowse (although it may be confusing).